src/terra-dev-site/example/ResponsiveListAndTableExample.jsx

```diff
@@ -0,0 +1,162 @@
+import IconAttachment from 'terra-icon/lib/icon/IconAttachment';
+import List, { Item } from 'terra-list';
+import ItemView from 'terra-clinical-item-view';
+import Table from 'terra-table';
+import classNames from 'classnames/bind';
+import styles from './ResponsiveListAndTableExample.scss';
+import mockData from './mock-data/table-select-data'
+
+//Replace ITEMviee.Display
+const display1 = <ItemView.Display icon={<IconPerson />} text="Asif Khan" />;
+const display2 = <ItemView.Display text="Care Position: Primary" />;
+const display3 = <ItemView.Display text="Room 100A" />;
+const display4 = <ItemView.Display text="Acuity: 5" />;
+const display5 = <ItemView.Display text="Start Time: 08-05-2016 12:00:00" />;
+const display6 = <ItemView.Display text="End Time: 08-05-2016 16:00:00" />;
+const comment = <ItemView.Comment text="Faint red rash appeared at 08-05-2016 13:24:00" />;
+const startAccessory = <IconAlert />;
+const endAccessory = <IconInformation />;
+
+const cx = classNames.bind(styles);
+
+const listDisplay = (
+  <List role="listbox">
+    <Item>
+      <ItemView
+        displays={[
+          <ItemView.Display icon={<IconPerson />} text="Asif Khan" textStyle="attention" />,
+          display2,
+          display3,
+          display4,
+          display5,
+          display6,
+        ]}
+        startAccessory={startAccessory}
+        comment={comment}
+        endAccessory={endAccessory}
+      />
+    </Item>
+    <Item>
+      <ItemView
+        displays={[
+          display1,
+          display2,
+          display3,
+          <ItemView.Display text="Acuity: 5" icon={<IconAttachment />} />,
+          display5,
+          display6,
+        ]}
+        startAccessory={startAccessory}
+        comment={comment}
+        endAccessory={endAccessory}
+        layout="twoColumns"
+      />
+    </Item>
+    <Item>
+      <ItemView
+        displays={[
+          display1,
+          display2,
+          display3,
+          display4,
+          display5,
+          display6,
+        ]}
+        reserveStartAccessorySpace
+        comment={comment}
+        endAccessory={endAccessory}
+      />
+    </Item>
+    <Item>
+      <ItemView
+        displays={[
+          display1,
+          display2,
+          <ItemView.Display text="Room 100A" textStyle="strikeThrough" />,
+          display4,
+          display5,
+          display6,
+        ]}
+        reserveStartAccessorySpace
+        comment={comment}
+        endAccessory={endAccessory}
+        layout="twoColumns"
+      />
+    </Item>
+    <Item>
+      <ItemView
+        displays={[
+          display1,
+          display2,
+          display3,
+          display4,
+          display5,
+          display6,
+        ]}
+        startAccessory={startAccessory}
+        comment={comment}
+      />
+    </Item>
+  </List>
+);
+
+
+const createCell = cell => ({ key: cell.key, children: cell.children, attrs: cell.attrs });
```

Comment:
Is createCell needed if this maps out with the same keys?

Reply:
Removed https://github.com/cerner/terra-ui/pull/310/commits/2f0433b8a6f6fb97139694a22a60e0ad1862bf40?file-filters%5B%5D=.jsx&file-filters%5B%5D=.mdx#diff-db8ae280acc81345eeba3a440c2c81285c34457faf9cbfb01de5508791e3283fL110